src/Microsoft.Identity.Web.TokenAcquisition/TokenAcquisition.cs

```diff
@@ -629,6 +629,39 @@ public async Task<AuthenticationResult> GetAuthenticationResultForAppAsync(
             }
         }
 
+        private static void ConfigureAtPopWithCredentials(
+            AcquireTokenForClientParameterBuilder builder,
+            IAppConfig appConfig,
+            TokenAcquisitionOptions tokenAcquisitionOptions,
+            IEnumerable<CredentialDescription>? clientCredentials,
+            bool sendX5C)
+        {
+            // Try to configure AtPop with custom signed assertion first
+            if (clientCredentials != null)
```

Comment:
It sounds like it's redoing what IdWeb does.

Reply:
Removed custom signed assertion configuration.
It will now be set up via delegate during CCA initialization.